cli/src/action/database/upgrade/node_id.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Provides the import action for node ids
+
+use std::path::PathBuf;
+
+use log::info;
+use splinter::node_id::store::file::FileNodeIdStore;
+use splinter::node_id::store::NodeIdStore;
+
+use crate::error::CliError;
+
+///Import node_id from one store to another
+///
+/// # Arguments
+///
+/// * `to` - the NodeIdStore receiving the node_id
+/// * `from` - the NodeIdStore supplying the node_id
+fn import_store(
+    to: &'_ dyn NodeIdStore,
+    from: &'_ dyn NodeIdStore,
+) -> Result<WarningEmited, CliError> {
+    match (from.get_node_id(), to.get_node_id()) {
+        (Ok(Some(id)), Ok(None)) => to
+            .set_node_id(id)
+            .map_err(|e| CliError::ActionError(format!("{}", e)))
+            .map(|_| WarningEmited::No),
+        (Ok(Some(_)), Ok(Some(_))) => Err(CliError::ActionError(
+            "destination store already has node_id set".to_string(),
+        )),
+        (Ok(None), _) => {
+            warn!("node_id file is empty, skipping import");
```

Comment:
Make this capital (and the the other warnings) to match the other logging that happens during the subcommand

Reply:
I think we should have all of the skipping logs start like "Skipping ...."